com.unity.shadergraph/ShaderGraphLibrary/ShaderVariablesFunctions.hlsl

```diff
@@ -45,19 +45,14 @@ bool IsPerspectiveProjection()
 }
 
 // Computes the world space view direction (pointing towards the viewer).
+float3 GetWorldSpaceViewDir(float3 positionWS)
```

Comment:
This is not ideal. Consider we want to have a custom hlsl shader for URP, now we have to include shader graph files, this should be in URP and shader graph include it.